APP-MANAGER

```diff
@@ -151,12 +155,43 @@ else
 	_appman
 fi
 
+# DOWNLOAD PORTABLE DEPENDENCES
+function _portable_dependences_setup() {
+	mkdir -p "$AMPATH"/bin
+
+	function _jq_installer() {
+		if command -v jq 1>/dev/null; then
+			amjq="jq"
+		else
+			amjq="$AMPATH/bin/jq"
+			jqversion=$(curl -Ls https://api.github.com/repos/jqlang/jq/releases | sed 's/[()",{}]/ /g; s/ /\n/g' | grep -o "https.*linux.*$jqarch" | head -1)
```

Comment:
where is Github api PAT Token?

Reply:
https://github.com/ivan-hc/AM/pull/533/commits/ea595ea0de99d5155e4793317cc9f3af0aae7afd

Reply:
add here line 167